website/static/js/filepage/revisions.js

```diff
@@ -90,6 +90,14 @@ var FileRevisionsTable = {
             });
         };
 
+
+        var popOver = function(element, isInit) {
+            if (!isInit) {
+                $(document.getElementById(element.id)).popover();
```

Comment:
You already have access to the DOM element in this function (`element`), so no need to get the element by ID.

```js
$(element).popover();
```